kata-deploy/scripts/kata-deploy.sh

```diff
@@ -57,10 +57,13 @@ function configure_crio() {
 	# backup the CRIO.conf only if a backup doesn't already exist (don't override original)
 	cp -n "$crio_conf_file" "$crio_conf_file_backup"
 
-	cat <<EOT | tee -a "$crio_conf_file"
+	grep -Fq '[crio.runtime.runtimes.kata-qemu]' $crio_conf_file \
```

Comment:
Nit: You could add variables for the multiple instances of `[crio.runtime.runtimes.kata-qemu]` and `[crio.runtime.runtimes.kata-fc]`.